src/cms/models/pages/page_translation.py

```diff
@@ -144,6 +144,29 @@ def combined_last_updated(self):
             return mirrored_page_translation.last_updated
         return self.last_updated
 
+    @property
+    def tags(self):
+        """
+        This functions returns a list of translated tags which apply to this function.
+        Supported tags:
+        * Live content: if the page of this translation has live content
+        * Empty: if the page contains no text
+
+        :return: A list of tags which apply to this translation
+        :rtype: list [ str ]
+        """
+        tags = []
+
+        is_mirrored = self.page.mirrored_page is not None
+        if is_mirrored:
+            tags.append(_("Live content"))
+
+        is_empty = not is_mirrored and len(self.text.strip()) == 0
+        if is_empty:
```

Comment:
```suggestion
        if not self.combined_text.strip():
```
Django already makes sure that `self.text` is always a string because it's defined that way in the model, and a string can only evaluate to `False` if it's length is 0, so it's safe to implicitly cast here.
Additionally, you could use `combined_text` here to handle the special case that live content is embedded, but the mirrored page is empty as well...